content/200-concepts/050-overview/300-prisma-in-your-stack/03-fullstack.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Fullstack'
+metaTitle: 'Building fullstack applications with Prisma'
+metaDescription: 'This page gives explains how to build fullstack applications with Prisma. It shows how Prisma fits in with fullstack frameworks and provides practical examples'
+---
+
+<TopBlock>
+
+Fullstack frameworks, such as Next.js and Sveltekit, blur the lines between the server and the client. These frameworks also provide different patterns for fetching and mutating data on the server.
+
+You can query your database using Prisma Client, using your framework of choice, from the server-side part of your application.
+
+</TopBlock>
+
+## Supported frameworks
+
+Here's a non-exhaustive list of frameworks and libraries you can use with Prisma:
+
+- [Next.js](https://nextjs.org/)
+- [Remix](https://remix.run)
+- [Sveltekit](https://kit.svelte.dev/)
+- [Nuxt](https://nuxt.com/)
+
+## Fullstack app example (e.g. Next.js)
+
+Assume you have a Prisma schema that looks similar to this:
+
+```prisma
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model Post {
+  id        Int     @id @default(autoincrement())
+  title     String
+  content   String?
+  published Boolean @default(false)
+  author    User?   @relation(fields: [authorId], references: [id])
+  authorId  Int?
+}
+
+model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+  posts Post[]
+}
+```
+
+You can now implement the logic for querying your database using [Prisma Client API](/concepts/components/prisma-client) inside `getServerSideProps`, `getStaticProps`, API routes, or using API libraries such as [tRPC](https://trpc.io/) and [GraphQL](https://graphql.org/).
+
+### `getServerSideProps`
```

Comment:
this doesn't get rendered in the right font size:

<img width="508" alt="image" src="https://user-images.githubusercontent.com/4058327/216575330-f5c10701-2bbf-4fea-a3e6-767d26551b99.png">

- you can the `<inlinecode>` html tag in headings instead, here is an example in the [reference](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#findfirst)